.changeset/forty-insects-lick.md

```diff
@@ -0,0 +1,9 @@
+---
+"wrangler": patch
```

Comment:
we sure this is a `patch` not `minor`. I'm always a bit confused, but these changes feel more like a minor to me?

Reply:
I'd call this a bug fix personally: absolutely no functional change and the intent is to fix flakey failures that users see.

Reply:
isn't the fact that we're adjusting the retry logic in case of API gateway failures a functional change? Or the fact that we're reducing the asset count?

Reply:
not pushing for either, mainly thinking out loud...and reconsidering all my changeset `minor` life choices 😅

Reply:
I'd personally say it's not a functional change as far as the user is concerned, no. But yeah, don't personally care at all and happy to be told I'm wrong!

Reply:
fair enough. 

> don't personally care at all and happy to be told I'm wrong
lol, same here

I'd just go ahead and merge as is